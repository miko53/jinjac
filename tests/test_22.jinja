{{ [ 1 , 2 , 3 , 50 ] | join('--') }} 
{{ [ 1 , 2 , 3 , 50 ] | join() }} 
{{ [ 1 , name , 3, 58.9, "testt", 0 ] | join(" ") }} 
