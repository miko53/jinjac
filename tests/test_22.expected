1--2--3--50 
12350 
1 mickael 3 58.900000 testt 0 
